ui/ui-components/components/alerts/Success.tsx

```diff
@@ -42,6 +42,7 @@ export default function SuccessAlert({
       show={show}
       onClose={() => setShow(false)}
       variant="success"
+      className="text-break"
```

Comment:
This is new to me and very powerful! https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

Reply:
Yep! And bootstrap's utility class adds support for IE/Edge as well https://getbootstrap.com/docs/4.4/utilities/text/#word-break